pyro/infer/inspect.py

```diff
@@ -510,11 +528,23 @@ def render_graph(
             # For sample_nodes - ellipse
             if node_data[rv]["distribution"]:
                 shape = "ellipse"
+                rv_label = rv
 
             # For param_nodes - No shape
             else:
                 shape = "plain"
-            cur_graph.node(rv, label=rv, shape=shape, style="filled", fillcolor=color)
+                rv_label = rv.replace(
+                    "$params", ""
+                )  # incase of neural network parameters
```

Comment:
Can you revert this change, since it's independent of the main goal of this PR? You're welcome to open a separate issue or PR with suggested changes to rendering settings like this one.

Reply:
Yes, but as I define `rv_label` in the `if` clause, it should also be defined in the `else` clause. Otherwise, it will be undefined

Reply:
You can just revert to using `rv`, once you revert this chance there's no need for `rv_label`.

Reply:
@r3v1 looks like this change did not get reverted